src/server_manager/messages/master_messages.json

```diff
@@ -1099,6 +1103,10 @@
     "message": "Set up Outline anywhere",
     "description": "This string appears in the server setup view as a header within a card. The card provides information about installing Outline on an arbitrary cloud server provider."
   },
+  "setup_beta": {
+    "message": "Beta",
```

Comment:
Drop "Beta" and use "Experimental" instead, so we are consistent

Reply:
Done

Reply:
I still see the message here. Delete?